src/views/Viewer.vue

```diff
@@ -1085,6 +1125,12 @@ export default {
 		cursor: pointer;
 	}
 
+	&__split {
```

Comment:
My BEM enthusiast brain is telling `&--split` would be more fitting :see_no_evil:

Reply:
Done :)